module/VuFindConsole/src/VuFindConsole/Command/Util/SwitchDbHashCommand.php

```diff
@@ -168,11 +168,11 @@ protected function getOpenSsl($algorithm)
      */
     protected function fixRow($row, ?BlockCipher $oldcipher, BlockCipher $newcipher): void
     {
-        $pass = ($oldcipher && $row['cat_pass_enc'] !== null)
-            ? $oldcipher->decrypt($row['cat_pass_enc'])
-            : $row['cat_password'];
-        $row['cat_password'] = null;
-        $row['cat_pass_enc'] = $pass === null ? null : $newcipher->encrypt($pass);
+        $pass = ($oldcipher && $row->getCatPassEnc() !== null)
+            ? $oldcipher->decrypt($row->getCatPassEnc())
+            : $row->getRawCatPassword();
+        $row->setRawCatPassword(null);
+        $row->setCatPassEnc($pass === null ? null : $newcipher->encrypt($pass));
```

Comment:
We can't change typehints to the entity interfaces yet because of the use of ->save() below. Addressing that will require more intrusive changes to this class, which I'll take care of as a future action.